packages/server/lib/cloud/protocol.ts

```diff
@@ -170,7 +174,31 @@ export class ProtocolManager implements ProtocolManagerShape {
   }
 
   async afterSpec () {
-    await this.invokeAsync('afterSpec', { isEssential: true })
+    const startTime = performance.now() + performance.timeOrigin
+
+    debug({ startTime })
```

Comment:
Should we have a description here around what this is?

Reply:
I think this debug can be removed- the important debug is on L191